mern/server/DAO/transcript_dao.js

```diff
@@ -0,0 +1,52 @@
+import Transcript from "../models/transcript.model.js";
+
```

Comment:
Basically the same comments as FlowDAO and QADAO.

Make sure to implement res.status correctly and change the constructor.